### PR TITLE
remove duplicates from scope names

### PR DESCRIPTION
**Pull request by ShefeeqPM**
fixing issue https://github.com/keycloak/keycloak/issues/24761

Removed duplicates from scopes as openid is specified twice.

### CODE REVIEW

File: services/src/main/java/org/keycloak/protocol/oidc/OIDCWellKnownProvider.java [hunk 1]
```diff
@@ -182,7 +182,7 @@ public Object getConfig() {
                     .map(ClientScopeModel::getName)
                     .collect(Collectors.toList());
             scopeNames.add(0, OAuth2Constants.SCOPE_OPENID);
-            config.setScopesSupported(scopeNames);
+            config.setScopesSupported(scopeNames.stream().distinct().collect(Collectors.toList()));
```
Comment (pedroigor) [inline services/src/main/java/org/keycloak/protocol/oidc/OIDCWellKnownProvider.java:185]: It should be simpler to just check if the set contains the scope and if so, don't add the scope.

**Review by pedroigor — CHANGES_REQUESTED** [review]
@ShefeeqPM Thanks. Can you please sign-off your commit (use `-s` option)?

File: services/src/main/java/org/keycloak/protocol/oidc/OIDCWellKnownProvider.java (hunk 1 above)
Comment (ShefeeqPM) [reply services/src/main/java/org/keycloak/protocol/oidc/OIDCWellKnownProvider.java]: this is addressed in another PR.
https://github.com/keycloak/keycloak/pull/26540.
kindly review that.

**Comment by jonkoops** [comment]
Closing this PR as it contains a bunch of unrelated commits. Please make sure your branch is clean and based on upstream `main`, then open a new PR.

**Pull request closed without merging**